web/src/main.tsx

```diff
@@ -46,6 +43,41 @@ if (isProduction) {
     },
   });
 }
+
+window.addEventListener('vite:preloadError', async (event: VitePreloadErrorEvent) => {
+  event.preventDefault();
+
+  try {
+    if ('serviceWorker' in navigator) {
+      const registrations = await navigator.serviceWorker.getRegistrations();
+      await Promise.all(registrations.map((r) => r.unregister()));
+    }
+
+    if ('caches' in window) {
+      const keys = await caches.keys();
+      await Promise.all(keys.map((key) => caches.delete(key)));
+    }
```

Comment:
What is this supposed to do?

I looked at both staging and production and `caches.keys()` contains the following:
```
0 : "workbox-precache-v2-https://app.electricitymaps.com/"
1 : "images"
2 : "icons"
```
I don't think we need to delete the last two?

But I suppose it don't hurt too much either?

Reply:
I think it's safer to remove them all, I believe it should be a rare case.